setup.py

```diff
@@ -1,27 +1,29 @@
-import setuptools
+import os
+
+from setuptools import find_packages, setup
 
 import versioneer
 
+rootpath = os.path.dirname(os.path.abspath(__file__))
+
 
-try:
-    with open('README.rst') as f:
-        long_description = f.read()
-except IOError:
-    long_description = ""
+def read(*parts):
+    return open(os.path.join(rootpath, *parts), "r").read()
 
 
-setuptools.setup(
-    name='stdlib-list',
-    license='MIT',
-    author='Jack Maney',
-    author_email='jackmaney@gmail.com',
-    url='https://github.com/jackmaney/python-stdlib-list',
+setup(
+    name="stdlib-list",
+    license="MIT",
+    author="Jack Maney",
+    author_email="jackmaney@gmail.com",
+    url="https://github.com/jackmaney/python-stdlib-list",
     version=versioneer.get_version(),
-    install_requires=['functools32;python_version<"3.2"'],
+    install_requires=["functools32;python_version<'3.2'"],
     extras_require={"develop": ["sphinx"]},
-    description='A list of Python Standard Libraries (2.6-7, 3.2-6).',
-    long_description=long_description,
+    description="A list of Python Standard Libraries (2.6-7, 3.2-8).",
+    long_description="{}".format(read("README.md")),
```

Comment:
I don't think you can do `"{}".format` and still install on Py2.6...

Reply:
... but I see other places in the code that does the same (e.g. `get_canonical_version`), and Travis is not testing on py2.6, and.. you know 2.6 :-)  I'm fine with dropping 2.6 support from docs since it most likely doesn't work anyway.

ps: all pydeps tests pass with this PR (not that it is exercising stdlib-list very deeply).

Reply:
Indeed ;-p

PS: I'm planning on issuing a release and then dropping 2.7, keeping the list of std for those pythons though may still be valuable.

Reply:
Please don't drop 2.7 until after the 2020-01-01 (still loads of users on 2.7).

Reply:
I like your optimism thinking that we will have another release before 2020-01-01 :smile: 

No worries about that  though ;-p